scripts/components/LviHomeProducts.ts

```diff
@@ -1,27 +1,64 @@
 import { getCombinedStyle } from '@smartface/extension-utils/lib/getCombinedStyle';
 import LviHomeProductsDesign from 'generated/my-components/LviHomeProducts';
+import GviProductItem from './GviProductItem';
+import store from 'store/index';
 const originalHeight = getCombinedStyle('.lviHomeProducts').height;
-
+import { Router } from '@smartface/router';
 export default class LviHomeProducts extends LviHomeProductsDesign {
-  pageName?: string | undefined;
-  constructor(props?: any, pageName?: string) {
-    // Initalizes super class for this scope
-    super(props);
-    this.pageName = pageName;
-  }
-  static getHeight(): number {
-    return originalHeight;
-  }
-  get showcaseTitle(): string {
-    return this.lblTitle.text;
-  }
-  set showcaseTitle(value: string) {
-    this.lblTitle.text = value;
-  }
-  get showcaseLinkText(): string {
-    return this.lblSeeAll.text;
-  }
-  set showcaseLinkText(value: string) {
-    this.lblSeeAll.text = value;
-  }
+    pageName?: string | undefined;
+    private __onProductClick: (product: any) => void;
+    private __items: any[] = [];
+    constructor(props?: any, pageName?: string) {
+        // Initalizes super class for this scope
+        super(props);
+        this.pageName = pageName;
+    }
+    static getHeight(): number {
+        return originalHeight;
+    }
+    get items(): any[] {
+        return this.__items;
+    }
+    set items(value: any[]) {
+        this.__items = value;
+        this.initGridView();
```

Comment:
we shouldn't be setting onItemBind everytime items get set. We need a initial function that sets binding probably inside constructor.